finality-aleph/src/sync/service.rs

```diff
@@ -112,7 +148,7 @@ where
                 justifications_from_user,
                 additional_justifications_from_user,
                 _block_requests_from_user,
-                _block_importer,
+                _block_importer: block_importer,
```

Comment:
I suspect you won't want that here, but rather within `Handler` in the future, so you could just pass `database_io` still packed there. But this can also be fixed in following PRs (or maybe I'm wrong), so not a blocker.

Reply:
whooops, fixed